packages/wonder-blocks-toolbar/docs.md

```diff
@@ -0,0 +1,106 @@
+> TODO(scottgrant): Finish building, and write more thorough docs.
+
+## Small toolbar example:
+
+```js
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const smallButton = <Button size="small">Small button</Button>;
+
+<Toolbar size="small" rightContent={[smallButton, smallButton]} />
```

Comment:
There should only ever be one "primary" button in a toolbar.  Can you change one of these to be a "secondary" button?

Reply:
Yep, will do.